unic/ucd/category/src/category.rs

```diff
@@ -321,41 +325,49 @@ impl GeneralCategory {
 impl GeneralCategory {
     /// `Lu` | `Ll` | `Lt`  (Short form: `LC`)
     pub fn is_cased_letter(&self) -> bool {
+        use self::abbr_names::*;
```

Comment:
I thought I explained it with the "or limited to the scope where it's needed" phrase. This is basically limiting the use of `*` to a small block of code, instead of one whole `impl`.